### PR TITLE
fix: add missing `globalParameters` field to `Query` type

### PR DESCRIPTION
**Pull request by novohatnya**

### CODE REVIEW

File: __schema__/globalParameters.graphql [hunk 1]
```diff
@@ -28,6 +30,8 @@ type GlobalParameters {
 
 """
 Input for `UpdateGlobalParameters` mutation.
+
+User must have scopes **global_parameters:write**
```
Comment (burlachenkovv) [inline __schema__/globalParameters.graphql]: It's a single scope, so maybe we should write `User must have a scope **global_parameters:write**`, like in the string above?

Comment (kiurchv) [reply __schema__/globalParameters.graphql]: Yes, you are right. Actually, scope always single, despite of number of allowances which it contains

Comment (novohatnya) [reply __schema__/globalParameters.graphql]: Сhanged it

**Comment by kiurchv** [comment]
I'm confused. Isn't we added `GlobalParameters` type earlier?

File: __schema__/globalParameters.graphql [hunk 2]
```diff
@@ -1,5 +1,7 @@
 """
-List of global parameters
+List of global parameters.
```
Comment (kiurchv) [inline __schema__/globalParameters.graphql]:
```suggestion
Represents global parameters of eHealth.
```

**Pull request merged**